project/templates/project/report_imper.html

```diff
@@ -0,0 +1,369 @@
+                                    <li>
+                                        <a class="fr-nav__link export-chart" href="#" data-chart-target="chart_waterfall" data-type="image/jpeg">JPEG</a>
+                                    </li>
+                                    <li>
+                                        <a class="fr-nav__link export-chart" href="#" data-chart-target="chart_waterfall" data-type="application/pdf">PDF</a>
+                                    </li>
+                                    <li>
+                                        <a class="fr-nav__link export-chart" href="#" data-chart-target="chart_waterfall" data-type="image/svg+xml">Vectoriel</a>
+                                    </li>
+                                </ul>
+                            </div>
+
+                            <button class="fr-btn fr-icon-drag-move-2-line fr-btn--tertiary fr-btn--sm fullscreen-chart" data-chart-target="chart_waterfall" data-toggle="tooltip" title="Afficher le graphique en plein écran"></button>
+                        </div>
+                        <div id="imper_nette_chart"></div>
+                    </div>
+                </div>
+            </div>
+
+            <div class="fr-notice fr-mt-4w">
+                <div class="fr-container">
+                    <div class="d-flex align-items-center justify-content-between">
+                        <div>
+                            <span class="fr-icon-information-line" aria-hidden="true"></span>
+                            <span class="fr-text--xs fr-mr-1w">Source de données: </span>
+                            <p class="fr-tag fr-tag--sm fr-tag--success">
+                                <strong>OCS GE</strong>
+                            </p>
+                        </div>
+                        <button class="fr-btn fr-btn--secondary fr-btn--sm fr-btn--icon-right fr-icon-arrow-down-s-fill mt-0" aria-expanded="false" aria-controls="target-data-2">Détails données et calcul</button>
+                    </div>
+                    <div class="fr-collapse" id="target-data-2">
+                        <h6 class="fr-mt-2w">Source</h6>
+                        <p class="fr-text--sm">Données d'OCcupation des Sols à Grande Echelle (OCS GE) de l'IGN, sur la période d'analyse traduite grâce à la matrice de passage.</p>
+
+                        <h6 class="fr-mt-2w">Calcul</h6>
+                        <p class="fr-text--sm">Imperméabilisation sur la période - Désimperméabilisation sur la période.</p>
+
+                        <h6 class="fr-mt-2w">Données</h6>
+                        {{ imper_nette_data_table }}
+                    </div>
+                </div>
+            </div>
+        </div>
+
+
+        <div class="fr-mt-7w">
+            <h2>Détails de l'Imperméabilisation entre {{ first_millesime }} et {{ last_millesime }}</h2>
+
+            <h3>Grandes familles de couverture des sols des surfaces artificialisées</h3>
```

Comment:
```suggestion
            <h3>Familles de couverture des sols des surfaces imperméabilisées</h3>
```
J'ai aussi remplacé `grande famille` par `famille` ici parce que les deux couvertures sont de la même grande famille